MerlinAU.sh

```diff
@@ -2574,8 +2637,8 @@ Please manually update to version $minimum_supported_version or higher to use th
     local checkChangeLogSetting="$(Get_Custom_Setting "CheckChangeLog")"
 
     if [ "$checkChangeLogSetting" = "ENABLED" ]; then
-        # Define the path to the log file
-        changelog_file="${FW_BIN_DIR}/Changelog-NG.txt"
+        #Files matching the patterns 'Changelog-NG.txt' or 'Changelog-386.txt'
+        changelog_file=$(find ${FW_BIN_DIR} -type f \( -name "Changelog-386.txt" -o -name "Changelog-NG.txt" \) -print | head -n 1)
```

Comment:
The "**find**" cmd statement uses the "**-type**" argument which is not valid with the built-in "find" command. It's valid with the Entware "find" utility which not all users may have installed. Also, I prefer to search for the pattern "**Changelog-*.txt**" which will find possible future changes in the name such as "Changelog-388.txt" in addition to current file names.